sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -61,6 +62,9 @@ struct sof_sdw_codec_info {
 		     struct snd_soc_dai_link *dai_links,
 		     struct sof_sdw_codec_info *info,
 		     bool playback);
+
+	bool late_probe;
```

Comment:
@RanderWang why do we need this book? Can we not simply check if the function pointer below is NULL?

Reply:
@ranj063  consider a case:  a codec has late_probe function but it is not integrated in some platforms, so we don't need to invoke this function.